components/common-data-items.js

```diff
@@ -254,6 +254,23 @@ export function SampleDataItems({
           </DataItemValue>
         </>
       )}
+      {item.protocols?.length > 0 && (
+        <>
+          <DataItemLabel>Protocols</DataItemLabel>
+          <DataItemList isCollapsible isUrlList>
+            {item.protocols.map((protocol) => (
+              <a
+                href={protocol}
+                key={protocol}
+                target="_blank"
+                rel="noopener noreferrer"
+              >
+                {protocol}
+              </a>
+            ))}
+          </DataItemList>
+        </>
+      )}
```

Comment:
If you have more than one, do the URLs end up without separators? Like `https://www.protocols.io/link1https://www.protocols.io/link2https://www.protcols.io/link3`? I feel like this needs SeparatedList.

Reply:
done